test/verify.test.js

```diff
@@ -307,77 +309,129 @@ test.serial('Verify "assets" is an Array of Object with a glob Arrays in path pr
   t.true(gitlab.isDone());
 });
 
-test('Throw SemanticReleaseError if "assets" option is not a String or an Array of Objects', async t => {
+test.serial('Throw SemanticReleaseError if "assets" option is not a String or an Array of Objects', async t => {
+  const owner = 'test_user';
```

Comment:
Why adding those intermediate variables?

Reply:
I added the `gitlab` because if you don't create it like for other test you get an extra `EINVALIDGLTOKEN` error, hence your test is not accurate (not checking only the `EINVALIDASSETS` error code is thrown. So I copied the same approach than the `Verify token and repository ...` tests to make sure that no other errors would be thrown.

But is you have another way to do it, I can correct it to that way.